workers/tests/integrationtests/test_oqt.py

```diff
@@ -105,19 +105,6 @@ def test_create_indicator_dataset_custom_fid_field_str(self):
         self.assertIsNotNone(indicator.result.description)
         self.assertIsNotNone(indicator.result.svg)
 
-    @oqt_vcr.use_cassette()
-    def test_create_indicator_from_database_invalid_arguments(self):
```

Comment:
shouldn't this test be replaced by another one?

Reply:
This test case tested if a Error got raised when feature and dataset+feature_id was given. This is now allowed. Not sure how to test the new behavior though.

Reply:
I added a test case where `create_indicator` is run providing `feature` and `dataset+feature_id` as arguments.